back/engines/commercial/id_nemlog_in/app/lib/id_nemlog_in/nemlog_in_verification.rb

```diff
@@ -4,10 +4,22 @@ module IdNemlogIn
   module NemlogInVerification
     include Verification::VerificationMethod
 
+    HOSTS_WITH_AGE_MINIMUM_15 = %w[kobenhavntaler.kk.dk]
+
     def verification_method_type
       :omniauth
     end
 
+    def entitled?(auth)
+      issuer = AppConfiguration.instance.settings['verification']['verification_methods'][0]['issuer']
+      return true unless HOSTS_WITH_AGE_MINIMUM_15.any? { |host| issuer.include?(host) }
```

Comment:
Why not just using `AppConfiguration.instance.host`? Something like: 
```suggestion
      return true unless AGE_REQUIREMENTS.key?(AppConfiguration.instance.host)
```
or
```suggestion
      return true unless AppConfiguration.instance.host.in?(HOSTS_WITH_AGE_MINIMUM_15)
```

Or am I missing something? 🧐

Reply:
Make sense. You're not missing anything, methinks.

Reply:
@adessy I like the idea of just using `AppConfiguration.instance.host`, but how can I test this.

If I just naively set the host in tests, then I get a route error:
```
context 'when verification method is used by Copenhagen' do
    before do
      configuration = AppConfiguration.instance
      configuration.host = 'https://kobenhavntaler.kk.dk'
      configuration.save!
    end
    
    => 1) IdNemlogIn::NemlogInOmniauth when verification method is used by Copenhagen does not verify an under 15-year-old
     Failure/Error: follow_redirect!
     
     ActionController::RoutingError:
       No route matches [GET] "/kobenhavntaler.kk.dk/auth/nemlog_in/callback"
```

Reply:
Try something like: 
```ruby
      allow(AppConfiguration).to receive(:instance)
        .and_return(instance_double(AppConfiguration, host: 'kobenhavntaler.kk.dk'))
```

Reply:
Thanks. I got it working with:
```
before do
      AppConfiguration.instance.update!(host: 'kobenhavntaler.kk.dk')
      host! 'kobenhavntaler.kk.dk'
    end
```